projects/js-packages/number-formatters/package.json

```
@@ -14,29 +14,36 @@
 		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Number Formatters"
 	},
 	"dependencies": {
-		"@babel/runtime": "^7",
 		"@wordpress/date": "5.19.0",
-		"debug": "4.4.0"
+		"debug": "4.4.0",
+		"tslib": "2.5.0"
 	},
 	"devDependencies": {
 		"@babel/core": "7.26.10",
 		"@babel/preset-react": "7.26.3",
 		"@jest/globals": "29.4.3",
+		"@knighted/duel": "2.1.3",
 		"@types/jest": "29.5.14",
 		"jest": "29.7.0",
 		"jest-environment-jsdom": "29.7.0",
+		"oxc-parser": "0.64.0",
 		"typescript": "5.8.2"
 	},
+	"type": "module",
+	"main": "./dist/cjs/index.cjs",
+	"module": "./dist/esm/index.js",
+	"types": "./dist/types/index.d.ts",
 	"exports": {
 		".": {
 			"jetpack:src": "./src/index.ts",
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
+			"types": "./dist/types/index.d.ts",
+			"import": "./dist/esm/index.js",
+			"require": "./dist/cjs/index.cjs"
 		}
 	},
 	"scripts": {
-		"build": "pnpm run clean && pnpm run compile-ts",
-		"clean": "rm -rf dist/",
+		"build": "pnpm run clean && duel --dirs",
+		"clean": "rm -rf dist",
 		"compile-ts": "tsc --pretty",
```

I'd probably do like this
```suggestion
		"build": "pnpm run clean && pnpm run compile-ts",
		"clean": "rm -rf dist",
		"compile-ts": "duel --dirs",
```
But this would also be fine.
```suggestion
		"build": "pnpm run clean && duel --dirs",
		"clean": "rm -rf dist",
```